### DISCUSSION TITLE
output qlog?

### DISCUSSION THREAD

**Discussion by Pieter-Bonten** [mitmproxy/mitmproxy#6778] [Q&A]
Dear.

For my master's thesis, I am currently researching TLS inspection possibilities in QUIC.
For this thesis, I am surveying the current state of TLS inspection in QUIC, and this is one of the pieces of software that I am comparing. 

Now QUIC connections are pretty difficult to interpret, compare and debug in wireshark. Because of this, there exists qlog and qvis. These are logging and visualisation formats for QUIC connections.

Is qlog logging currently available in this software? If not, is it being considered an upcoming feature?

**Comment by mhils** [comment]
We're using aioquic under the hood, so getting qlog should not be too hard. We don't have explicit qlog support in mitmproxy yet.
